frontend/kannada-explorer/kannada.css

```
@@ -0,0 +1,297 @@
+    --kannada-text: #f5f7ff;
+    --kannada-mute: #d0d8ea;
+    --kannada-soft: #aab8d2;
+    --kannada-glow: rgba(255, 159, 28, 0.3);
+}
+
+body.kannada-main {
+    background:
+        radial-gradient(circle at top left, rgba(255, 191, 0, 0.15), transparent 28%),
+        linear-gradient(135deg, #0f172a 0%, #111827 45%, #172033 100%);
+    color: var(--kannada-text);
+    font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)